pkg/providers/instancetype/suite_test.go

```diff
@@ -218,10 +219,6 @@ var _ = Describe("Instance Types", func() {
 			v1alpha1.LabelInstanceGPUCount:                     "1",
 			v1alpha1.LabelInstanceGPUMemory:                    "16384",
 			v1alpha1.LabelInstanceLocalNVME:                    "900",
-			v1alpha1.LabelInstanceAcceleratorName:              "t4",
-			v1alpha1.LabelInstanceAcceleratorManufacturer:      "nvidia",
-			v1alpha1.LabelInstanceAcceleratorCount:             "1",
-			v1alpha1.LabelInstanceAcceleratorMemory:            "16384",
```

Comment:
Don't we only need to remove this line?

Reply:
No, since the groups of instances that have accelerators and those that have GPUs are distinct, there's no way to combine all the labels and satisfy a single instance type

Reply:
Ah I see, can we make one suite_test for the `LabelInstanceGPU*` labels like you have here and one with the `LabelInstanceAccelerator*` for completeness then?